docs/content/onedrive.md

```diff
@@ -106,6 +112,23 @@ To copy a local directory to an OneDrive directory called backup
 
     rclone copy /home/source remote:backup
 
+### OneDrive for Business ###
+
+There is experimental support for OneDrive for Business.
+Select "b" when ask
+```
+Choose OneDrive account type?
+ * Say b for a OneDrive business account
+ * Say p for a personal OneDrive account
+b) Business
+p) Personal
+b/p> 
+```
+After that rclone requires two authentications. First to authenicate your account
+and second to get the final token to access your companies resources.
+
+Headless authentication is not working at the moment.
```

Comment:
You should probably change this to "Headless authentication is not working at the moment with rclone authorize - you'll have to use the copy the config file method".